### DISCUSSION TITLE
loading tracks from maker2jbrowse

### DISCUSSION THREAD

**Discussion by matthewlfabian** [GMOD/jbrowse-components#3291] [Q&A]
Hi - I have JBrowse Desktop & a FASTA file loaded. I have a .GFF that I can overlay as a single track, but I want to load it into JBrowse as separate tracks, separated by feature type (e.g. transcripts vs. gene models). Using command-line jbrowse, I ran "maker2jbrowse" to convert the .GFF, generated via MAKER, into separate tracks for JBrowse. The output is a folder containing "names", "seq", & "tracks" subfolders. The "tracks" subfolders look correct, i.e. separate folders for the indvidual MAKER components. Does anyone know how to load this data into JBrowse Desktop? I've attached an image of one of the subfolders' contents.

![Screenshot (1)](https://user-images.githubusercontent.com/116763086/198133472-09f630fa-95f6-4bd2-964f-f21b409b9d31.png)

**Comment by cmdcolin** [comment]
I could brainstorm but there might not be a super easy way to do this with jbrowse2 desktop. you might even have a better time with jbrowse1 desktop at the moment. we should probably make an equivalent maker2jbrowse for jbrowse 2!

**Reply by matthewlfabian** [reply]
Hi - Thanks for the suggestion. I downloaded JBrowse 1 Desktop, added the FASTA, & opened up the maker2jbrowse output as a "data directory", which worked like a charm!
